src/pages/Clients/Eth2/Nimbus.tsx

```diff
@@ -75,7 +75,7 @@ export const NimbusDetails = ({ shortened }: { shortened?: boolean }) => (
           <Link
             external
             primary
-            to="https://status-im.github.io/nim-beacon-chain/validator.html#key-management"
+            to="https://status-im.github.io/nimbus-eth2/validator.html#key-management"
```

Comment:
@zah
Is this page WIP?
I saw

![image](https://user-images.githubusercontent.com/9263930/95445962-41475080-0992-11eb-9a8e-502af6ae1bf7.png)